src/python_bring_api/bring.py

```diff
@@ -293,3 +297,58 @@ def removeItem(self, listUuid: str, itemName: str) -> Response:
         except RequestException as e:
             _LOGGER.error(f'Exception: Could not remove item {itemName} from list {listUuid}:\n{traceback.format_exc()}')
             raise BringRequestException(f'Removing item {itemName} from list {listUuid} failed due to request exception.') from e
+
+
+    def notify(self, listUuid: str, notificationType: str, itemName: str = None) -> Response:

```

Comment:
I would set the notification type to a union of allowee values instead of str.

Reply:
was thinking maybe an enum or somthing like that would be a better solution but i don't have that much python experience, so i would appreciate it if you could rewrite this part

Reply:
PR for changes: https://github.com/tr4nt0r/python-bring-api/pull/1